src/Sylius/Bundle/ShippingBundle/composer.json

```diff
@@ -31,25 +31,27 @@
         "php": "^8.2",
         "stof/doctrine-extensions-bundle": "^1.4",
         "sylius/money-bundle": "^2.0",
-        "sylius/resource-bundle": "^1.10",
+        "sylius/resource-bundle": "^1.10 || dev-symfony-7",
         "sylius/shipping": "^2.0",
         "symfony/clock": "^6.4.0 || ^7.0",
         "symfony/framework-bundle": "^6.4.1 || ^7.0"
     },
     "conflict": {
-        "doctrine/orm": "2.15.2",
+        "doctrine/orm": ">= 3.0.0",
         "stof/doctrine-extensions-bundle": "1.8.0"
     },
     "require-dev": {
         "doctrine/orm": "^2.13",
-        "matthiasnoback/symfony-dependency-injection-test": "^4.2",
+        "matthiasnoback/symfony-dependency-injection-test": "^5.1",
         "phpspec/phpspec": "^7.2",
         "phpunit/phpunit": "^9.5",
         "polishsymfonycommunity/symfony-mocker-container": "^1.0",
         "symfony/browser-kit": "^6.4.0 || ^7.0",
         "symfony/dependency-injection": "^6.4.1 || ^7.0",
         "symfony/form": "^6.4.1 || ^7.0",
-        "symfony/validator": "^6.4.0 || ^7.0"
+        "symfony/validator": "^6.4.0 || ^7.0",
+        "friendsofsymfony/rest-bundle": "^3.0",
+        "jms/serializer-bundle": "^5.0"
```

Comment:
They could be added here in alphabetical order

Reply:
done